From e9c1a13c45c11e4d142b886472a28e5dacfb5edc Mon Sep 17 00:00:00 2001
From: Patrick Tjin <pattjin@google.com>
Date: Tue, 5 Dec 2017 14:55:14 -0800
Subject: [PATCH 085/420] PM: wakeup_reasons: Fix formatting for printk

current output looks like this:

[  133.772467] Abort: Wakeup IRQ 730 wdog-bark pending[  133.777343] PM: suspend exit 2017-11-30 19:05:34.974168313 UTC

Change it to this:

[  133.772467] Abort: Wakeup IRQ 730 wdog-bark pending
[  133.777343] PM: suspend exit 2017-11-30 19:05:34.974168313 UTC

Change-Id: Ic38338a3274b83256788077db6f1040605b79603
Signed-off-by: Patrick Tjin <pattjin@google.com>
---
 kernel/power/wakeup_reason.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/power/wakeup_reason.c b/kernel/power/wakeup_reason.c
index 20d218a73cae..44d8da2952c5 100644
--- a/kernel/power/wakeup_reason.c
+++ b/kernel/power/wakeup_reason.c
@@ -161,7 +161,7 @@ static void print_wakeup_sources(void)
 	const struct list_head *wakeups;
 
 	if (suspend_abort) {
-		pr_info("Abort: %s", abort_reason);
+		pr_info("Abort: %s\n", abort_reason);
 		return;
 	}
 
-- 
2.15.0

